humble/distribution.yaml

```diff
@@ -6109,7 +6109,7 @@ repositories:
   schunk_svh_library:
     doc:
       type: git
-      url: https://github.com/fzi-forschungszentrum-informatik/schunk_svh_library.git
+      url: https://github.com/SCHUNK-GmbH-Co-KG/schunk_svh_library.git
```

Comment:
This change is fine; no problem to move the repository.

But can I suggest that you actually move it from the `fzi-forschungszentrum-informatik` organization into the `SCHUNK-GmbH-Co-KG` organization?  That way GitHub will automatically do redirects.  Copying it like this makes it confusing for anyone using the old URL (it will still be there, just not be updated).

That said, this is not a blocking comment so if you can't/don't want to do this, we can proceed.

Reply:
+1 for the move which will enable redirects.

Reply:
Though we should still update the urls so I'm going to merge all of these.